op-batcher/batcher/sync_actions.go

```diff
@@ -36,15 +54,17 @@ func (s syncActions) String() string {
 // range to load into the local state, and whether to clear the state entirely. Returns an boolean indicating if the sequencer is out of sync.
 func computeSyncActions[T channelStatuser](newSyncStatus eth.SyncStatus, prevCurrentL1 eth.L1BlockRef, blocks queue.Queue[*types.Block], channels []T, l log.Logger) (syncActions, bool) {
 
+	m := l.With("newSyncStatus", summarizeSyncStatusNumbers(newSyncStatus))
```

Comment:
nit, it would be nicer if you added 4 log fields instead, like
```suggestion
	m := l.With("head_l1", newSyncStatus.HeadL1, ...)
```
since we have a structured logger. It may also be beneficial to log the hashes as well, not only the numbers. Above example should display something like `0x12..ab:123` for `head_l1` because the sync status fields are `{L1,L2}BlockRef`s

Reply:
[92543b5](https://github.com/ethereum-optimism/optimism/pull/14563/commits/92543b5a2bdeb07bc9df8306e9620f5ec29755cb)